beacon-chain/blockchain/receive_block.go

```diff
@@ -57,7 +57,8 @@ func (s *Service) ReceiveBlock(ctx context.Context, block interfaces.ReadOnlySig
 	}
 
 	// Handle post block operations such as pruning exits and bls messages.
-	if err := s.prunePostBlockOperationPools(ctx, blockRoot); err != nil {
+	// Prune service config vlaues based on the block copy if it's the head.
```

Comment:
What does this mean? Maybe Prune block operations if it's the head

Reply:
can rephrase